src/main/sd/sd.h

```diff
@@ -0,0 +1,58 @@
+#ifndef MEXT2_SD_H
+#define MEXT2_SD_H
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+#define __packed __attribute__((packed))
```

Comment:
I don't see a point to define __packed every time we want to declare some struct. I suppose we could create some separate module like 'common_utils' or something in which we could keep useful defines like that

Reply:
__packed moved to common/common.h file